This is a testharness.js-based test.
FAIL supportsSession rejects when options not supported promise_test: Unhandled rejection with value: object "ReferenceError: promise_rejects is not defined"
Harness: the test ran to completion.

